templates/plugin-gruntfile.mustache

```diff
@@ -11,10 +11,11 @@ module.exports = function( grunt ) {
 			options: {
 				textdomain: '{{textdomain}}',
 			},
-			target: {
-				files: {
-					src: [ '*.php', '**/*.php', '!node_modules/**', '!php-tests/**', '!bin/**' ]
-				}
+			update_all_domains: {
```

Comment:
What's the significance of changing this from `target` to `update_all_domains`? Will it have an impact on execution?